libs/execution_context/ebpf_program.c

```diff
@@ -1547,6 +1563,9 @@ _ebpf_program_test_run_work_item(_Inout_opt_ void* work_item_context)
         goto Done;
     }
 
+    // Issue: https://github.com/microsoft/ebpf-for-windows/issues/1844
+    // Running at elevated IRQL for long periods of time can cause the system to hang.
+    // This should periodically lower the IRQL to allow other work to be done.
```

Comment:
nit: should we raise and lower the irql after every iteration of program run? To measure the run time, we can capture start / end time for each iteration and add them?

Reply:
Planning on fixing this as part of https://github.com/microsoft/ebpf-for-windows/issues/1844